test/integration/narrow/routes/utils/user-needs-notification.test.js

```diff
@@ -0,0 +1,60 @@
+const { userNeedsNotification } = require('../../../../../app/routes/utils/user-needs-notification')
```

Comment:
Should this be in the integration section of the tests? I was expecting to see it in the unit section - as an aside, the term integration is a little loosely used in our repos I think.

Reply:
the folder structure for our tests makes no sense to me. I would also describe this as a 'unit' test, but all the other tests for the `routes/utils` folder are here in `integration/narrow/routes/utils` so I figured, for now at least, it's best that all the files testing a folder are in the same folder 🤷